### DISCUSSION TITLE
git lfs pull

### DISCUSSION THREAD

**Discussion by gitrimius** [shanteacontrols/OpenDeck#42] [Q&A]
Hi,
I was trying to clone the OpenDeck repository with following guide:

> Cloning OpenDeck repository
> 
> OpenDeck repository uses several Git sub-modules. Therefore, running git clone url won't work since those sub-modules won't be cloned. Use the following command to clone an entire repository with sub-modules:
> 
> git clone --recursive https://github.com/paradajz/OpenDeck.git
> git lfs pull
> Several binaries are stored with Git LFS, hence the git lfs pull command.

When entering git lfs pull in terminal I get following message:

> <pre>git: &apos;lfs&apos; is not a git command. See &apos;git --help&apos;.
> 
> The most similar command is
> 	log
> </pre>

Any idea how to get around this?
Thanks!

**Comment by paradajz** [comment]
The answer is in required packages.

**Comment by gitrimius** [comment]
ok, I had run all those commands and they gave no error.
To be sure I redid the procedure and when running git lfs pull I get the following message:

> osboxes@osboxes:~$ git lfs pull
> Not in a git repository.

**Reply by paradajz** [reply]
See the updated command

**Comment by gitrimius** [comment]
Hi, I have run the command in the OpenDeck folder and it resulted in no errors now...
I am really new to all this...
